instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/tracers/graphql_trace.rb

```diff
@@ -98,11 +98,15 @@ def analyze_query(query:, &block)
 
           def execute_query(query:, &block)
             attributes = {}
-            attributes['graphql.operation.name'] = query.selected_operation_name if query.selected_operation_name
-            attributes['graphql.operation.type'] = query.selected_operation.operation_type
+            operation_type = query.selected_operation.operation_type
+            operation_name = query.selected_operation_name
+
+            attributes['graphql.operation.name'] = operation_name if operation_name
+            attributes['graphql.operation.type'] = operation_type
```

Comment:
There's a MUST-level part of the spec requiring `operation.type` to be one of `query`, `mutation` or `subscription`. We weren't checking it previously. I'm not certain we need to validate it now. 

Will the gem only send those operation types? What are your thoughts @karmingc?